### PR TITLE
Added  popups & modals, data table elements

### PR DESCRIPTION
**Pull request by AkioFaris**
related issues from jdi-light project:
[1807](https://github.com/jdi-testing/jdi-light/issues/1807), [1808](https://github.com/jdi-testing/jdi-light/issues/1808), [1809](https://github.com/jdi-testing/jdi-light/issues/1809), [1810](https://github.com/jdi-testing/jdi-light/issues/1810), [1811](https://github.com/jdi-testing/jdi-light/issues/1811), [1812](https://github.com/jdi-testing/jdi-light/issues/1812), [1813](https://github.com/jdi-testing/jdi-light/issues/1813)

### CODE REVIEW

File: angular-site/src/app/data-table/paginator/paginator-configurable-example.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * @title Configurable paginator
+ */
+@Component({
+  selector: 'paginator-configurable-example',
+  templateUrl: 'paginator-configurable-example.html',
+  styleUrls: ['paginator-configurable-example.css'],
+})
+export class PaginatorConfigurableExample {
```
Comment (mf-code) [inline angular-site/src/app/data-table/paginator/paginator-configurable-example.ts:12]: add new line before

File: angular-site/src/app/data-table/sort-header/sort-overview-example.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+/**
+ * @title Sorting overview
+ */
+@Component({
+  selector: 'sort-overview-example',
+  templateUrl: 'sort-overview-example.html',
+  styleUrls: ['sort-overview-example.css'],
+})
+export class SortOverviewExample {
```
Comment (mf-code) [inline angular-site/src/app/data-table/sort-header/sort-overview-example.ts:20]: new line , and so on formatting issues in another places. try to use js code style formatter (you can start from here https://habr.com/ru/post/220229/ I guess)

Comment (AkioFaris) [reply angular-site/src/app/data-table/sort-header/sort-overview-example.ts:20]: According to the [Angular coding style guide](https://angular.io/guide/styleguide) new line is not needed here. Do you think I should add it nevertheless?

File: angular-site/src/app/data-table/paginator/paginator-configurable-example.ts (hunk 1 above)
Comment (AkioFaris) [reply angular-site/src/app/data-table/paginator/paginator-configurable-example.ts:12]: According to the [Angular coding style guide](https://angular.io/guide/styleguide) new line is not needed here. Do you think I should add it nevertheless?

**Pull request merged**